extensions/adobe/experience/campaign/offer.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/offer",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title":
+    "An offer that is shown as part of a Marketing Campaign. This could be personalized as per the user profile or a generalized offer.",
+  "type": "object",
+  "description":
+    "This represents the instance of an offer displayed in a message.",
+  "definitions": {
+    "offer": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/offer/id": {
+          "title": "Offer ID",
+          "type": "integer",
+          "description": "Identifier of the offer.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/offer/name": {
+          "title": "Offer Name",
+          "type": "string",
+          "description": "Name of the offer."
+        },
+        "https://ns.adobe.com/experience/campaign/offer/activityId": {
+          "title": "Activity ID",
+          "type": "integer",
+          "description":
+            "Identifier of the offer activity in the Offer Service. Offer Activity is a container which may have multiple offers and a subset of them will be shown to the user based on their profile attributes or rules set by the Marketeer. ",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/offer/activityName": {
+          "title": "Activity Name",
+          "type": "string",
+          "description": "Name of the offer activity in the Offer Service."
+        },
+        "https://ns.adobe.com/experience/campaign/offer/placementId": {
```

Comment:
`placement/id`

Reply:
Also, if you can use `uri` references, they are much better than `integer` references.

Reply:
:+1: for `placement/id`
not sure if I can use uri refs here with our current implementation. I'd like to keep it this way for now.